src/controllers/bookmarkController.js

```diff
@@ -0,0 +1,62 @@
+import BookmarkModel from '../helpers/bookmarkModel';
+import errorResponse from '../helpers/index';
+/**
+ * @description class representing Bookmark Controller
+ * @class BookmarkController
+ */
+class BookmarkController {
+  /**
+   * @description - This method is responsible for bookmarking an article
+   * @static
+   * @param {object} request - Request sent to the router
+   * @param {object} response - Response sent from the controller
+   * @returns {void}
+   * @memberof BookmarkController
+   */
+  static async bookmark(request, response) {
+    const { slug } = request.params;
+    const userId = request.userData.payload.id;
+    await BookmarkModel.addBookmark(response, slug, userId);
+  }
+
+  /**
+   * @description - This method is responsible for getting a user's bookmarks
+   * @static
+   * @param {object} request - Request sent to the router
+   * @param {object} response - Response sent from the controller
+   * @returns {object} - object representing response message
+   * @memberof BookmarkController
+   */
+  static async getAllBookmark(request, response) {
+    try {
+      const userId = request.userData.payload.id;
```

Comment:
If you're not using userId other than here in this method, why not destructure the id from request.userData.payload and pass it.......

Reply:
OK, that make sense. I will do that straight up.

Reply:
You didn't fix this....

Reply:
I am using the userId, It is a reference to a column on the bookmark table. I destructured the id, as you suggested, and I renamed it to userId so I can make reference to the userId column in the database.